src/goto-programs/remove_unreachable.cpp

```diff
@@ -55,3 +55,9 @@ void remove_unreachable(goto_programt &goto_program)
       it->make_skip();
   }
 }
+
+void remove_unreachable(goto_functionst &goto_functions)
```

Comment:
For completeness I happily note that this is in 8397d11acfc of #961 - thank you, @martin-cs!